lib/iruby/kernel.rb

```diff
@@ -109,7 +109,7 @@ def error_message(e)
     def complete_request(msg)
       # HACK for #26, only complete last line
       code = msg[:content]['code']
-      if start = code.rindex("\n")
+      if start = code.rindex(/\s|\R/)
```

Comment:
What does `\R` do? i couldn't find it in the regex [docs](http://ruby-doc.org/core-2.2.0/Regexp.html#class-Regexp-label-Character+Classes).